cassandra-dac/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java

```
@@ -796,4 +798,137 @@ public Response insertRecordWithTTL(
     logQueryElapseTime("insertRecordWithTTL", startTime);
     return response;
   }
+
+  @Override
+  public Response getRecordsByIdsWithSpecifiedColumnsAndTTL(
+      String keyspaceName,
+      String tableName,
+      Map<String, Object> primaryKeys,
+      Map<String, String> propertiesWithAlias,
+      Map<String, String> ttlPropertiesWithAlias) {
+    long startTime = System.currentTimeMillis();
+    ProjectLogger.log(
+        "CassandraOperationImpl: getRecordsByIdsWithSpecifiedColumnsAndTTL call started at "
+            + startTime,
+        LoggerEnum.INFO);
+    Response response = new Response();
+    try {
+
+      Selection selection = QueryBuilder.select();
+
+      if (MapUtils.isNotEmpty(propertiesWithAlias)) {
+        propertiesWithAlias
+            .entrySet()
+            .stream()
+            .peek(
+                p ->
+                    ProjectLogger.log(
+                        "key=" + p.getKey() + " value=" + p.getValue(), LoggerEnum.INFO))
+            .forEach(
+                property -> {
+                  if (StringUtils.isBlank(property.getValue())) {
+                    selection.column(property.getKey());
+                  } else {
+                    selection.column(property.getKey()).as(property.getValue());
+                  }
+                });
+      }
+
+      if (MapUtils.isNotEmpty(ttlPropertiesWithAlias)) {
+        ttlPropertiesWithAlias
+            .entrySet()
+            .stream()
+            .peek(
+                p ->
+                    ProjectLogger.log(
+                        "key=" + p.getKey() + " value=" + p.getValue(), LoggerEnum.INFO))
+            .forEach(
+                property -> {
+                  if (StringUtils.isBlank(property.getValue())) {
+                    selection.ttl(property.getKey());
```

We should return an error if no alias is provided for ttl.